synthetic_signwriting/hands/hands.py

```diff
@@ -0,0 +1,67 @@
+import math
+import numpy as np
+
+from scipy.spatial.transform import Rotation
+
+
+def create_left_hand(pose: np.ndarray):
+    min_p = np.min(pose, axis=0)  # Get the maximum point in the pose
+
+    # multiply the x-axis by -1 to reflect the hand
+    left_hand = pose
+    left_hand[:, 0] = -left_hand[:, 0]
+
+    diff = min_p - np.min(left_hand, axis=0)
+    left_hand += diff
+
+    return left_hand
+
+
+def get_hand_rotation(pose: np.ndarray):
+    point_1 = pose[0]  # Wrist
+    point_2 = pose[9]  # Middle CMC
+    vec = point_2 - point_1
+
+    return 90 + math.degrees(math.atan2(vec[1], vec[0]))
+
+
+def rotate_hand(pose: np.ndarray, angle: float, axis='z'):
```

Comment:
for rotation and scaling, use the `PoseNormalizer`

see example: 
https://github.com/sign-language-processing/segmentation/blob/d54f2c46b28156844a3841a59b3d14aae47b2604/sign_language_segmentation/figures/optical_fow/optical_flow.py#L46-L53